spark/src/test/java/org/apache/iceberg/TestDataFileSerialization.java

```diff
@@ -68,14 +69,16 @@
 
   private static final Map<Integer, Long> VALUE_COUNTS = Maps.newHashMap();
   private static final Map<Integer, Long> NULL_VALUE_COUNTS = Maps.newHashMap();
+  private static final Map<Integer, Long> NAN_VALUE_COUNTS = Maps.newHashMap();
   private static final Map<Integer, ByteBuffer> LOWER_BOUNDS = Maps.newHashMap();
   private static final Map<Integer, ByteBuffer> UPPER_BOUNDS = Maps.newHashMap();
 
   static {
     VALUE_COUNTS.put(1, 5L);
     VALUE_COUNTS.put(2, 3L);
+    VALUE_COUNTS.put(4, 2L);
     NULL_VALUE_COUNTS.put(1, 0L);
-    NULL_VALUE_COUNTS.put(2, 2L);
```

Comment:
why are we modifying this line instead of adding a new line?

Reply:
Oh good catch, I guess I thought I duplicated the line myself but seems like it was from the original code; I'll add it back